src/content/configuration/extending-configurations.mdx

```diff
@@ -0,0 +1,122 @@
+---
+title: Extends
+sort: 12
+contributors:
+  - burhanuday
+---
+
+## extends
+
+`string | string[]`
+
+<Badge text="webpack v5.82.0+" />
+<Badge text="webpack-cli v5.0.3+" />
```

Comment:
How does this look?

```suggestion
<Badge text="webpack v5.82.0+" /> <Badge text="webpack-cli v5.0.3+" />
```

Reply:
the same basically
<img width="827" alt="image" src="https://user-images.githubusercontent.com/25934701/236656587-c74687d4-b28c-4c46-a084-1887a5867dd0.png">